### PR TITLE
Support Microsoft.AspNetCore for net 4.6.1 and up

### PR DESCRIPTION
**Pull request by StefH**

### CODE REVIEW

File: src/WireMock.Net/Owin/AspNetCoreSelfHost.cs [hunk 1]
```diff
@@ -39,7 +39,7 @@ public AspNetCoreSelfHost([NotNull] WireMockMiddlewareOptions options, [NotNull]
 
             _logger = options.Logger ?? new WireMockConsoleLogger();
 
-            foreach (string uriPrefix in uriPrefixes)
+            foreach (string uriPrefix in uriPrefixes) // .Select(u => u.EndsWith("/") ? u : $"{u}/")) // Always append an / at the end.
```
Comment (chrischu) [inline src/WireMock.Net/Owin/AspNetCoreSelfHost.cs]: Is the code supposed to be commented out?

File: src/WireMock.Net/Owin/OwinSelfHost.cs [hunk 2]
```diff
@@ -26,7 +27,7 @@ public OwinSelfHost([NotNull] WireMockMiddlewareOptions options, [NotNull] param
 
             _logger = options.Logger ?? new WireMockConsoleLogger();
 
-            foreach (string uriPrefix in uriPrefixes)
+            foreach (string uriPrefix in uriPrefixes) // .Select(u => u.EndsWith("/") ? u : $"{u}/")) // Always append an / at the end.
```
Comment (chrischu) [inline src/WireMock.Net/Owin/OwinSelfHost.cs]: Is the code supposed to be commented out?

File: test/WireMock.Net.Tests/RequestWithPathTests.cs [hunk 3]
```diff
@@ -14,6 +14,36 @@ public class RequestWithPathTests
     {
         private const string ClientIp = "::1";
 
+        // [Fact] : TODO : this test fails???
```
Comment (chrischu) [inline test/WireMock.Net.Tests/RequestWithPathTests.cs]: Probably remove comment and TODO?

File: src/WireMock.Net/Owin/AspNetCoreSelfHost.cs (hunk 1 above)
Comment (StefH) [reply src/WireMock.Net/Owin/AspNetCoreSelfHost.cs]: Yes should be removed.

File: src/WireMock.Net/Owin/OwinSelfHost.cs (hunk 2 above)
Comment (StefH) [reply src/WireMock.Net/Owin/OwinSelfHost.cs]: Yes should be removed.

File: test/WireMock.Net.Tests/RequestWithPathTests.cs (hunk 3 above)
Comment (StefH) [reply test/WireMock.Net.Tests/RequestWithPathTests.cs]: I keep this for now as reminder,

**Comment by StefH** [comment]
@chrischu Thanks for your review. I've updated some code and commented on your comments. Can you please approve?

File: test/WireMock.Net.Tests/RequestWithPathTests.cs (hunk 3 above)
Comment (chrischu) [reply test/WireMock.Net.Tests/RequestWithPathTests.cs]: But the test doesn't fail anymore right?

Comment (StefH) [reply test/WireMock.Net.Tests/RequestWithPathTests.cs]: Somehow this test still fails. Also for netcore.
So for now i've removed the test.

**Comment by chrischu** [comment]
Since it is already merged: Is there a plan for creating a new release with these changes?

File: test/WireMock.Net.Tests/RequestWithPathTests.cs (hunk 3 above)
Comment (chrischu) [reply test/WireMock.Net.Tests/RequestWithPathTests.cs]: Well then the underlying problem (that WireMock.Net does not handle encoded URLs correctly) still persists :(.

**Comment by StefH** [comment]
I'll create a new version today.

**Comment by StefH** [comment]
1.0.4.11 NuGet created. Please test.
And if it still fails, please create an issue with some steps to reproduce it.

**Pull request merged**